ocpp-v1_6/src/main/java/eu/chargetime/ocpp/JSONServer.java

```diff
@@ -53,8 +54,8 @@ public JSONServer(ServerCoreProfile coreProfile) {
         featureRepository.addFeatureProfile(coreProfile);
     }
 
-    public void enableWSS(SSLContext sslContext) {
-        listener.enableWSS(sslContext);
+    public void enableWSS(WssFactoryBuilder wssFactoryBuilder) {
```

Comment:
This is an exposed API, which means that changes will break peoples code. I suggest that we keep the original one and add this as an overload.

Reply:
Good point. Updated class to be backward compatible.